charts/clickhouse/templates/clickhouse-operator/configmaps/etc-configd-files.yaml

```diff
@@ -36,7 +36,6 @@ data:
             <engine>Engine = MergeTree PARTITION BY event_date ORDER BY event_time TTL event_date + interval {{ .Values.clickhouseOperator.queryLog.ttl }} day</engine>
             <flush_interval_milliseconds>{{ .Values.clickhouseOperator.queryLog.flushInterval }}</flush_interval_milliseconds>
         </query_log>
-        <query_thread_log remove="1"/>
```

Comment:
is this change intended?

Reply:
Yes, I enabled the `query_thread_log` with 7 days to better understand how many threads are used for each kind of operation. If 7 days is too much then we will reduce it further.